common-jvm/src/main/java/org/ergoplatform/mosaik/AppMosaikRuntime.kt

```diff
@@ -143,12 +140,19 @@ abstract class AppMosaikRuntime(
     }
 
     override fun showError(error: Throwable) {
-        // check if this error is on first app load, we show a retry button in this case
-        if (viewTree.content == null) {
-            noAppLoaded(error)
-        } else {
-            super.showError(error)
-        }
+        showDialog(
+            MosaikDialog(
+                getUserErrorMessage(error),
+                "OK",
+                null,
+                null,
+                null
```

Comment:
It's better to use named arguments in such cases

Reply:
Removed them